source/data-model-partials/data-model.md

```diff
@@ -0,0 +1,99 @@
+## Data model diagram
+This diagram was correct as of 27th February 2018, and is already slightly out of date. It should be useful to see the 
+overall structure of things though. 
+ 
+ You can click the image to open it bigger.
+![alt text](/images/eff-data-model-2019-02-27.svg "Data model of the Ethnicity Facts and Figures Publisher, as of 27th February 2019.")
+
+[View source file for data model diagram](https://drive.google.com/file/d/1atJKUx9wbMmeOOCkgdjt9WmnCGOE9r5B/view)
+
+## Site structure
+The Ethnicity Facts and Figures website is organised as a hierarchy (tree) in four levels:
+
+**Level 1: Home page**
+ 
+The home page has a template in the Publisher app, but is not modelled explicitly by any record in the database.  
+It is the conceptual root of the site hierarchy, and is the parent of all *Topics*.
+
+**Level 2: Topics**
+
+Topics are the highest level grouping of pages on the site, and are listed on the home page as links. They live in the 
+`topic` table in the database. Clicking a topic link from the homepage takes you to the topic page, which lists all
+*Subtopics* that belong to the topic.
+
+**Level 3: Subtopics**
+
+Subtopics live in the `subtopic` table of the database, and belong to exactly one topic, defined by the `topic_id`
+field. Each subtopic belonging to a topic has an accordion section on the topic page, which can be opened to reveal the
+*Measures* that belong to the subtopic.
+
+**Level 4: Measures and their versions**
+
+Measures are the individual leaf nodes of the site hierarchy and live in the `measure` table of the database. Measures 
+can have multiple versions, and so the majority of data for displaying measure pages lives in the `measure_version`
+table, with only minimal data applicable to all versions of a measure in the `measure` table itself.
+
+The database structure allows for measures to belong to multiple subtopics (via the join table `subtopic_measure`), but
+there is no user interface to allow this to happen and in practice all measures belong to one subtopic only.
+
+Measure versions are really the main feature of our data model. They are the things that are edited by users of the 
+Publisher, and the thing that become the main content of the website - the pages of commentary, data, tables and charts.
+
+Measure versions have zero or more dimensions (stored in the `dimension` table). In practice all measure versions have 
+at least one dimension.
+
+**Dimensions**
+Dimensions show a particular cut of the data being presented in the measure, and are displayed one after another on our
```

Comment:
Maybe avoid "displayed one after another on our measure pages" which is super presentation-specific, and just say "all displayed on the measure page". So that when we change to table-of-contents, this isn't immediately out of date?